<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionDBSet"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCtrlDialogStaion"/>
    <addaction name="separator"/>
    <addaction name="actionCtrlMsgPro"/>
    <addaction name="actionCtrlMsgRcv"/>
    <addaction name="actionCtrlMsgFind"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetMsgPro">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>解算</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetStation">
   <property name="windowTitle">
    <string>测站</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMsgRcv">
   <property name="windowTitle">
    <string>接收</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMsgFind">
   <property name="windowTitle">
    <string>查找</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="actionDBSet">
   <property name="text">
    <string>数据库设置</string>
   </property>
   <property name="toolTip">
    <string>数据库设置</string>
   </property>
  </action>
  <action name="actionCtrlDialogStaion">
   <property name="text">
    <string>测站窗口</string>
   </property>
  </action>
  <action name="actionCtrlMsgPro">
   <property name="text">
    <string>解算窗口</string>
   </property>
  </action>
  <action name="actionCtrlMsgRcv">
   <property name="text">
    <string>接收窗口</string>
   </property>
  </action>
  <action name="actionCtrlMsgFind">
   <property name="text">
    <string>查找窗口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
